cirq-core/cirq/sim/state_vector_test.py

```diff
@@ -175,7 +175,7 @@ def test_sample_no_indices_repetitions():
 
 @pytest.mark.parametrize('use_np_transpose', [False, True])
 def test_measure_state_computational_basis(use_np_transpose: bool):
-    linalg.can_numpy_support_shape = lambda s: use_np_transpose
+    linalg.can_numpy_support_shape = lambda shape: use_np_transpose
```

Comment:
nit: use `_shape` instead since the arg is unused?
```suggestion
    linalg.can_numpy_support_shape = lambda _shape: use_np_transpose
```
Same thing for several other places below.

Reply:
The actual mypy complaint was about the replacement function not matching the previous signature as the `can_numpy_support_shape` has `shape` argument.  Looking at it again, this test overwrites `cirq.linalg. can_numpy_support_shape` without restoring it back.

This is fixed in f7da1617ae3617c0bc5a4134c538ef3772b49529.